### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by nickbalestra**

### CODE REVIEW

File: test/utils.spec.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    const template = {compiler:[7,'>= 4.0.0'],main:function(){return'Hello world!';},useData:!0}; // eslint-disable-line
+    const result = utils.validator(template, 'handlebars');
+
+    test('should be valid', () => {
+      expect(result.isValid).toBe(true);
+    });
+  });
+
+  describe('when component\'s template type is jade', () => {
+    const result = utils.validator('', 'jade');
```
Comment (matteofigus) [inline test/utils.spec.js]: Why the utils.validate signature has even the type param? This module should just be about handlebars?

File: package.json [hunk 2]
```diff
@@ -29,7 +29,9 @@
   "devDependencies": {
     "eslint": "^3.14.0",
     "eslint-config-opentable": "^6.0.0",
-    "ghooks": "^2.0.0"
+    "eslint-plugin-import": "^1.16.0",
+    "ghooks": "^2.0.0",
```
Comment (matteofigus) [inline package.json]: what is ghooks for?

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  include:
+    - node_js: "4"
+    - node_js: "5"
+    - node_js: "6"
+    - node_js: "7"
+
+env:
+  global:
+  - secure: OUooygPgokU2pa5FmVKb5tEQ+NjrUS7vr2al2MYqQDi7VOViOkKiP+1TMKDaPEo3+H3BQlsRd/HxItSCucg5Wy+45pMsTnYgR0Lsp2zmcuHmY67EPOM3B6bdCyPtB5VokrpO3O+uBTtMPAKS/sQq62gKV5Xq/K1t5CQ31EgfCd4=
+  - secure: P/+5dvk3zwkDnUmNCEX8fgNkLAQ5s6j+dR4h3Zn5UJBBUKSZaz1QbfEX+uZ5Rn4l4aRGdvG2PGStySiS/MHmPKhWJfMMEkSFjERLUF07U2gLSkn1UOQeh0zBWIb5TehwIuuTNPmOoSE/7xgmXwF83Dq8ALjDCmo6VrtoZes9E2g=
```
Comment (matteofigus) [inline .travis.yml]: Why do we need the secure params?

**Review by matteofigus — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (nickbalestra) [reply package.json]: git hooks. 
using it to run test before commit :)

File: .travis.yml (hunk 3 above)
Comment (nickbalestra) [reply .travis.yml]: we don't 👍

**Comment by nickbalestra** [comment]
fixes #2 #1

**Review by matteofigus — APPROVED** [review]

**Pull request merged**